### PR TITLE
Increasing privacy policy compliance

### LINKED ISSUE #9050: Increase privacy policy compliance (for Huawei and everyone really)
I took the opportunity with 2.15.3 release to also (finally) publish 2.15.x on to the Huawei AppConnect app channel

I noticed (and they notified us) that we are not fully compliant with their privacy policy (section 7):

https://developer.huawei.com/consumer/en/doc/30202#h1-1613641610376-6

Specifically I think in the first Help page we probably need a new sub-menu for "Privacy" and in there we need to put a link to our existing privacy policy (we use https://ankiweb.net/account/terms as ours also) with some extra information about opt-in and fully anonymous nature of our analytics and our crash reporting system

Also note that this is a good opportunity to do an audit of the data we collect to make sure that it is truly anonymized and does not contain any PII (Personally Identifiable Information) so that GDPR compliance is as easy as it should be (that is: there is no way to request what information we have on you or delete it because it's completely anonymous so we could not locate you if we tried and there is therefore nothing to see or delete)

### PR DESCRIPTION
**Pull request by tanmayChakrawarty**
## Purpose / Description
Increasing privacy policy compliance for everyone.

## Fixes
Fixes  #9050

## How Has This Been Tested?

Tested on physical device moto g30 (Android 11).

## Output

https://user-images.githubusercontent.com/75254780/121690167-1f830800-cae3-11eb-9e2d-d470f0db0e20.mp4

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented on your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/res/values/03-dialogs.xml [hunk 1]
```diff
@@ -244,6 +244,12 @@
     <string name="help_item_facebook">Facebook</string>
     <string name="help_item_twitter">Twitter</string>
 
+    <string name="help_title_privacy">Privacy</string>
+    <string name="help_item_ankidroid_privacy_policy">AnkiDroid Privacy Policy</string>
+    <string name="help_item_ankiweb_privacy_policy">AnkiWeb Privacy Policy</string>
+    <string name="help_item_terms_and_conditions">Terms and Conditions</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values/03-dialogs.xml:250]: AnkiWeb

File: AnkiDroid/src/main/res/values/constants.xml [hunk 2]
```diff
@@ -196,6 +196,10 @@
     <string name="resetpw_url">http://ankiweb.net/account/resetpw</string>
     <string name="register_url">https://ankiweb.net/account/register</string>
     <string name="link_reddit">https://www.reddit.com/r/Anki/</string>
+    <string name="link_ankidroid_privacy_policy">https://github.com/ankidroid/Anki-Android/wiki/Privacy-Policy</string>
+    <string name="link_ankiweb_privacy_policy">https://ankiweb.net/account/privacy</string>
+    <string name="link_terms_and_conditions">https://ankiweb.net/account/terms</string>
+    <string name="link_crash_reports">https://github.com/ankidroid/Anki-Android/wiki/Crash-Reports</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values/constants.xml:202]: This isn't useful information for an end user, I feel it is explained by our privacy policy, but if not then we need a user facing page

Vote to remove this

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.java [hunk 3]
```diff
@@ -83,6 +83,12 @@ public static DialogFragment createInstance(Context context) {
                         new LinkItem(R.string.help_item_discord, R.drawable.discord, UsageAnalytics.Actions.OPENED_DISCORD, R.string.link_discord),
                         new LinkItem(R.string.help_item_facebook, R.drawable.facebook, UsageAnalytics.Actions.OPENED_FACEBOOK, R.string.link_facebook),
                         new LinkItem(R.string.help_item_twitter, R.drawable.twitter, UsageAnalytics.Actions.OPENED_TWITTER, R.string.link_twitter)
+                ),
+                new ItemHeader(R.string.help_title_privacy, R.drawable.ic_baseline_privacy_tip_24, UsageAnalytics.Actions.OPENED_PRIVACY,
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/HelpDialog.java:87]: Order: we want ankidroid things before AnkiWeb things. Ankidroid is mandatory, AnkiWeb is optional

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Looks good! Few bits and we can get this in

**Comment by david-allison-1** [comment]
Just want to say thanks for the video, made this so much easier to look through!

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Also a few unit tests which need updating

**Comment by david-allison-1** [comment]
Unit tests still need updating

File: AnkiDroid/src/main/res/values/constants.xml (hunk 2 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/res/values/constants.xml]: agreed

File: AnkiDroid/src/main/res/values-af/03-dialogs.xml [hunk 4]
```diff
@@ -232,6 +232,10 @@
     <string name="help_item_twitter">Twitter</string>
     <string name="help_title_send_exception">Send troubleshooting report</string>
     <string name="help_dialog_exception_report_debounce">Report already submitted</string>
+    <string name="help_title_privacy">Privacy</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values-af/03-dialogs.xml:235]: Please revert this change.

> Add the string to the first file in the list (/values) and no other file.

See: https://github.com/ankidroid/Anki-Android/wiki/Development-Guide#adding-translations

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]
This is really fantastic, more comprehensive than I was thinking even (I guess I didn't realize how much data could really be flying around!) and looks great. Thanks

**Comment by mikehardy** [comment]
Hi there!

Apologies for the delay, and if you're already submitted something to OpenCollective for PRs merged in June 2021 PRs merged, you may feel free to ignore this and it will be processed shortly

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for June 2021 submissions

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

(I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged in May)

Thanks!

**Pull request merged**